src/modals/SettingsModal/GeneralSettingsSection.tsx

```diff
@@ -31,6 +34,20 @@ const GeneralSettingsSection = () => {
     updateSettings
   } = useGlobalContext()
 
+  const [requiresPasswordConfirmation, setRequiresPasswordConfirmation] = useState<boolean>(false)
```

Comment:
No need to be so explicit with the `boolean` type, since you are passing `false` as initial value.

Reply:
It's habit (preference) to well-type things as I go. It prevents bugs since the type checker will know what I intend...